### ISSUE TITLE
Crash on app launch, java.lang.NoClassDefFoundError: Failed resolution of: Lcom/android/vending/billing/IInAppBillingService$Stub;

### ISSUE THREAD

**Issue by acb** [dooboolab/react-native-iap#1554]
### Version of react-native-iap
7.5.0

### Version of react-native
0.63.3

### Platforms you faced the error (IOS or Android or both?)
Android

### Expected behavior
App launches

### Actual behavior
App crashes on launch.  `adb logcat` yields the message

```
java.lang.NoClassDefFoundError: Failed resolution of: Lcom/android/vending/billing/IInAppBillingService$Stub;
```

### Tested environment (Emulator? Real Device?)
Real device, android 11.  Ran the app with `npx react-native run-android`

### Steps to reproduce the behavior
Add react-native-iap to your project and attempt to start your app.

**Comment by acb** [comment]
I'm pretty sure that this was my fault for not installing the billing library properly, I'll close it.

**Comment by acb** [comment]
Never mind, I'm still getting the same error with that installed.  Losing my mind over here.

**Comment by MarcosHY** [comment]
Any news? We had version 4.5.2 and we had to update it, so we used 7.5.1: we tested it and it was ok, but now in production crashes have skyrocketed but only in Android 11, and we cannot replicate them on Android studio or our real devices. We also see no logs in Sentry or Google Play Console (at least yet) that fit that behaviour.
The changes in the release are only some changes in our ReactNative code, so that can't be the issue, specially when it works on every other device than an Android 11; and the only library changed is this IAP one, so we believe the error must be here
